### PR TITLE
Fix: support get helm chart values from which index.yaml urls is not …

### PR DESCRIPTION
**Pull request by caiqi1111**
…completed

### Description of your changes

Support get  values from helm repository which index.yaml entries url is not completed . 

image

Fixes #

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/utils/helm/helm_helper.go [hunk 1]
```diff
@@ -345,7 +345,13 @@ func (h *Helper) GetValuesFromChart(repoURL string, chartName string, version st
 	var urls []string
 	for _, chartVersion := range chartVersions {
 		if chartVersion.Version == version {
-			urls = chartVersion.URLs
+			for _, url := range chartVersion.URLs {
+				if !(strings.HasPrefix(url, "https://") || strings.HasPrefix(url, "http://")) {
+					urls = append(urls, fmt.Sprintf("%s/%s", repoURL, url))
```
Comment (nuclearwu) [inline pkg/utils/helm/helm_helper.go:350]: which type of helm repo needs to splice repourl and url? consider to add comments

**Review by wonderflow — COMMENTED** [review]
good catch, please also add tests

File: pkg/utils/helm/helm_helper.go (hunk 1 above)
Comment (caiqi1111) [reply pkg/utils/helm/helm_helper.go:350]: I deploy a harbor according https://github.com/goharbor/harbor/releases, it need to splice repourl and url.

**Comment by caiqi1111** [comment]
@wonderflow I add the test.

**Comment by Somefive** [comment]
@caiqi1111 Could you try to rebase the master branch and repush?

**Comment by caiqi1111** [comment]
@Somefive ok, I did it

**Review by Somefive — APPROVED** [review]

**Pull request merged**